src/Elmish.WPF/DynamicViewModel.fs

```diff
@@ -8,6 +8,23 @@ open Microsoft.Extensions.Logging
 
 open BindingVmHelpers
 
+/// Represents all necessary data used to create a binding.
+type Binding<'model, 'msg> =
+  internal
+    { Name: string
+      Data: BindingData<'model, 'msg> }
+
+
+[<AutoOpen>]
+module internal Helpers =
```

Comment:
Why move this?

Reply:
Because it uses the `Binding` type and it's used in this file